media/materials/scripts/gazebo.material

```diff
@@ -1515,7 +1515,7 @@ fragment_program Gazebo/WideLensMapFS glsl
     param_named c2 float 1
     param_named c3 float 0
     param_named f float 1
-    param_named fun float3 0 1
+    param_named fun float3 0 0 1
```

Comment:
This one might not change anything, but it seemed odd to not complete the default float3 values

Reply:
If `float3 0 1` was working before, wouldn't Ogre have been interpreting it as `float3 0 1 0`? This looks like it would change behavior from using `atan(param)` to using `param`.

Reply:
I agree, this is suspicious

Reply:
👍